src/App.tsx

```diff
@@ -3,16 +3,21 @@ import Home from "./pages/Home";
 import SignUp from "./pages/SignUp";
 import SignIn from "./pages/SignIn";
 import ForgotPassword from "./pages/ForgotPassword";
+import Theme from "./theme";
```

Comment:
```suggestion
import theme from "./theme";
```
so this is how import/export works:
```
// export default, since you only have one thing that will be the default component to be imported from the file
Example:
in TestComponent.js
export const TestComponent = () => {
  return (
    <div>hello!</div>
)};

export default TestComponent;
-- end of file --


in App.js
import TestComponent from './TestComponent';

const App = () => {
  return (
     <div>
         <TestComponent />
     </div>
  )
};

export default App;
-- end of file --


// export component without default, when you have multiple things in the file, but you want to import only some things
Example:
in TestComponent.js
const someText = "hello! this is a text that's not supposed to be exported, and should be used only in this component";
export const textExportTest = "hello! this is a text that can be exported";
export const TestComponent = () => {
  return (
    <div>{someText}</div>
)};
-- end of file --

in App.js
import { TestComponent, textExportTest } from './TestComponent';

const App = () => {
  return (
     <div>
         <TestComponent />
        {textExportTest}
     </div>
  )
};

export default App;
-- end of file --
```

https://medium.com/@etherealm/named-export-vs-default-export-in-es6-affb483a0910

Reply:
^ let me know if it's still unclear!

Reply:
It's starting to make sense, I will try the edits you suggested before our meeting today. Thank you!

Reply:
I added an import { ThemeProvider } to App.tsx and I added an empty export to the d.ts file. Now everything is compiling. 

I'm not sure if this is complete and the 'styled' from 'styled-components' is not being called yet due to the code not being written for it, or if I need to do something further.

Let me know! New commits incoming for review. :)